docs/internals/ARCHITECTURE.md

```diff
@@ -51,7 +51,7 @@ Replicas execute committed prepares in sequence number order by applying a batch
 local state. Because all replicas start with the same (empty) state and the state transition function
 is deterministic, the replicas arrive at the same state.
 
-If a primary fails, the consensus algorithm proper ensures that a different replica becomes a
```

Comment:
This emphasis is intentional, to underscore the difference between replication (fast path) and view change (actual consensus, slow path)